core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -726,6 +726,14 @@ public enum Property {
       "The number of threads on each tablet server available to retrieve"
           + " summary data, that is not currently in cache, from RFiles.",
       "2.0.0"),
+  TSERV_LAST_LOCATION_MODE("tserver.last.location.mode", "compact", PropertyType.LAST_LOCATION_MODE,
+      "Describes how the system will assign tablets initially by defining how the 'last' location is updated."
+          + " If 'compact' is the mode, then the system will assign tablets based on the data locality (e.g. the last compaction location)."
+          + " If 'assign' is the mode, then tablets will be initially assigned to the last place they were assigned."
+          + " If 'unload' is the mode, then tablets will be initially assigned to the last place they were unloaded from (i.e. requires a clean shutdown)."
```

Comment:
```suggestion
          + " If 'unload' is the mode, then tablets will be initially assigned to the last place they were unloaded from (i.e. requires a clean shutdown)."
```

I'm not sure we need this one at all. This would require a clean shut down in order to freeze the current assignments. And, in the case of that clean shut down, this would be no different than the `assign` mode. If there is not a clean shutdown, then the field would merely be blank... which isn't helpful.

Reply:
ok, that is reasonable argument.  I was briefly thinking that one would want only one or the other to be able to see the discrepency between the last location and the new locations when scanning the metadata and using the "unload" mode.